llvm/lib/IR/Attributes.cpp

```diff
@@ -526,13 +526,9 @@ std::string Attribute::getAsString(bool InAttrGrp) const {
 
   if (hasAttribute(Attribute::UWTable)) {
     UWTableKind Kind = getUWTableKind();
-    if (Kind != UWTableKind::None) {
-      return Kind == UWTableKind::Default
-                 ? "uwtable"
-                 : ("uwtable(" +
-                    Twine(Kind == UWTableKind::Sync ? "sync" : "async") + ")")
-                       .str();
-    }
+    assert(Kind != UWTableKind::None &&
```

Comment:
I'm not sure if the `Kind` is always expected to not be `UWTableKind::None`, but the current code seems to make that assumption. If it is None, we fall through to the [llvm_unreachable at the bottom of the function](https://github.com/llvm/llvm-project/blob/c87a7b3bdb673747f2242ba2edc7d5b2f5b53c30/llvm/lib/IR/Attributes.cpp#L643).